docs/pages/faq.mdx

```diff
@@ -104,9 +104,9 @@ Here are the major versions of Teleport and their support windows:
 
 | Release | Release Date     | EOL          | Minimum `tsh` version|
 |---------|------------------|--------------|----------------------|
+| v14.0   | Sep 18, 2023     | June 2024    | v13.x.x              |
```

Comment:
```suggestion
| v14.0   | Sep 18, 2023     | September 2024    | v13.x.x              |
```

v14 is out of support when v17 is released, and there's now 4 months between majors, so v17 is due a year after v14.

Reply:
Fixed in 6318d42392f2ee1323c6fe373d1db3c4f31c4062. Also added a more up-to-date description of our release cadence.